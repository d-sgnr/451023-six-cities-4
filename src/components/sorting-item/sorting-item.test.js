import React from "react";
import renderer from "react-test-renderer";
import SortingItem from "./sorting-item.jsx";

it(`SortingItem should be rendered correctly`, () => {
  const tree = renderer.create(
      <SortingItem
        title={`Popular`}
        activeSortType={`Popular`}
        onSortingChange={() => {}}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
