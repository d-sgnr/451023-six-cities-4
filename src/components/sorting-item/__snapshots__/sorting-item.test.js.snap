// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingItem should be rendered correctly 1`] = `
<li
  className="places__option places__option--active"
  onClick={[Function]}
  tabIndex="0"
>
  Popular
</li>
`;
